packages/codegen/generate.condo.server.utils.js

```diff
@@ -0,0 +1,155 @@
+const { get, isEmpty, isObject, isArray } = require('lodash')
+
+const conf = require('@open-condo/config')
+const { GQLError } = require('@open-condo/keystone/errors')
+const { makeRemoteExecutor } = require('@open-condo/keystone/stitchSchema')
+const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
+
+
+const IS_DEBUG = conf.NODE_ENV === 'development' || conf.NODE_ENV === 'test'
+
+const CONDO_API_URL = `${conf.CONDO_DOMAIN}/admin/api`
+const DEFAULT_LOCALE = conf.DEFAULT_LOCALE
+const CONDO_ACCESS_TOKEN_KEY = conf.CONDO_ACCESS_TOKEN_KEY
+const ACCEPT_LANGUAGE = conf.ACCEPT_LANGUAGE
+const CONDO_REFRESH_TOKEN_KEY = conf.CONDO_REFRESH_TOKEN_KEY
```

Comment:
What happens if these values ​​are empty? Should we throw an error?

Reply:
Use default values (same as in consts before) for this case